src/internal/packager/helm/chart.go

```diff
@@ -54,6 +54,10 @@ type InstallUpgradeOptions struct {
 	Timeout time.Duration
 	// Retries for the helm install/upgrade
 	Retries int
+	// Pkg is the zarf package being installed
+	Pkg *v1alpha1.ZarfPackage
```

Comment:
.Metdata.Name is the only field used from the package currently. Are you thinking that you might want other fields from the package in the future? Maybe a `string` or `v1alpha1.ZarfMetadata` would be more appropriate.

Reply:
Possibly, more of a future proofing in case we want to do more package based labeling in the future. I don't have a specific use case for it now though so I don't mind narrowing the scope of that variable

Reply:
I guess we can use the package name and the namespace override label if it exists to look up the package later! So I can trim this down to be just the name